fix hardcoding of 'gcc', add optimization flags defined in $CFLAGS
author: Kenneth Hoste (HPC-UGent)
--- npstat/trunk/Makefile.orig	2016-02-15 01:16:51.000000000 +0100
+++ npstat/trunk/Makefile	2016-03-10 09:25:46.000000000 +0100
@@ -1,8 +1,7 @@
 all: npstat
 
-
 npstat: 
-	gcc -o npstat NPStat-v0.99.c -lgsl -lgslcblas -lm
+	${CC} ${CFLAGS} -o npstat NPStat-v0.99.c -lgsl -lgslcblas -lm
 
 clean: 
-	rm npstat
\ No newline at end of file
+	rm npstat
